docs/development.rst

```diff
@@ -25,13 +25,7 @@ Setup
 
 Crypto-cpp
 ----------
-
-By default, the library uses ``crypto-cpp`` and so it must be configured properly for development:
-
-1. Compile it from sources (https://github.com/starkware-libs/crypto-cpp)
-2. Copy ``libcrypto_c_exports.dylib`` from ``crypto-cpp/src/starkware/crypto/ffi/libcrypto_c_exports.dylib`` to ``starknet_py/utils/crypto``
-
-To use python implementation instead, ``DISABLE_CRYPTO_C_EXTENSION`` environment variable can be set to ``false``
+By default, the library uses ``crypto-cpp``. To use python implementation instead, ``DISABLE_CRYPTO_C_EXTENSION`` environment variable can be set to ``false``
```

Comment:
`crypto-cpp-py` and provide a link

Reply:
Done, sorry I forgot to add this straight away 🙈